spring-integration-hazelcast/src/test/java/org/springframework/integration/hazelcast/leader/LeaderInitiatorTests.java

```diff
@@ -227,7 +227,7 @@ public TestCandidate candidate() {
 		@Bean
 		public Config hazelcastConfig() {
 			Config config = new Config();
-			config.getCPSubsystemConfig().setCPMemberCount(3)
+			config.getCPSubsystemConfig().setCPMemberCount(0)
```

Comment:
Yes, you are right: with HZ-3.12 it "does not start in unsafe mode with less than 3 nodes".
So, this CP Member `0` is something new in HZ 4.0.
Thank you for sharing that and probably it's is good to revise all the tests to work in that unsafe mode when we really don't need a distributed consensus to satisfy 😄